include/NovelRT/Utilities/Misc.h

```diff
@@ -1,19 +1,21 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
 
-#ifndef NOVELRT_NOVELUTILITIES_H
-#define NOVELRT_NOVELUTILITIES_H
-#include <algorithm>
+#ifndef NOVELRT_UTILITIES_MISC_H
+#define NOVELRT_UTILITIES_MISC_H
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
 
-namespace NovelRT {
-class NovelUtilities {
-public:
-  static const int SDL_SUCCESS = 0;
-  static inline const std::string CONSOLE_LOG_GENERIC = "NovelRT";
-  static inline const std::string CONSOLE_LOG_APP = "Application";
-  static inline const std::string CONSOLE_LOG_GFX = "GFX";
-  static inline const std::string CONSOLE_LOG_AUDIO = "Audio";
-  static inline const std::string CONSOLE_LOG_INPUT = "Input";
-};
+namespace NovelRT::Utilities {
+  class Misc {
+  public:
+    static const int SDL_SUCCESS = 0;
+    static inline const std::string CONSOLE_LOG_GENERIC = "NovelRT";
```

Comment:
`inline const`?

Reply:
That was not me but I'll rectify, lol.

Reply:
Uh, hold on, maybe I am misunderstanding. What is wrong with this?

Reply:
Why is it both `inline` and `const`?

Reply:
Ah, I see; its a new C++ 17 feature.

Reply:
Okay, marking as resolved then.